src/goto-instrument/goto_program2code.cpp

```diff
@@ -110,7 +110,7 @@ void goto_program2codet::scan_for_varargs()
         to_side_effect_expr(r).get_statement() == ID_va_start)
       {
         assert(r.has_operands());
```

Comment:
This assertion is now unnecessary (`to_unary_expr` already does this)

Reply:
removed